cobigen-templates/templates-devon4j/src/main/templates/crud_openapi_ionic_client_app/templates.xml

```diff
@@ -3,26 +3,26 @@
   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="2.1">
 
   <templates>
-    <templateExtension ref="${variables.etoName#lower_case}-detail.page.ts" mergeStrategy="tsmerge"/>
-    <templateExtension ref="${variables.etoName#lower_case}-list.page.ts" mergeStrategy="tsmerge_override"/>
+    <templateExtension ref="${variables.etoName#lower_case}-detail.page.ts" mergeStrategy="tsmerge"/>
+    <templateExtension ref="${variables.etoName#lower_case}-list.page.ts" mergeStrategy="tsmerge_override"/>
```

Comment:
I think you forgot to pull the latest changes from CobiGen master. This file is outdated, it should not be modified.